pwndbg/commands/__init__.py

```diff
@@ -23,10 +23,26 @@
 
 commands = []
 
+def list_current_commands():
+    gdb.execute('set pagination off')
+    command_list = gdb.execute('help all', False, True).strip().split('\n')
```

Comment:
Btw in `execute('help all', False, True)` the `False, True` part is not so explicit. Can we pass those as keyword arguments so it is immediately known what are we setting in here?